SPA vs web (#1958)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/quickstart/register-app/cli/register-app.md

1. Enter the command: `okta apps create`.
2. Give your app a name.

> 3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application will be using the tokens in Angular, React, Vue, or other browser-side code.  Pick **Web** if your application will be using the tokens solely in server-side code.

Reviewer: ```suggestion
3. Select **SPA** or **Web** as the **Type of Application**. Select **SPA** if your application is using the tokens in Angular, React, Vue, or other browser-side code.  Pick **Web** if your application is using the tokens solely in server-side code.
```
